.eslintignore

```diff
@@ -27,8 +27,8 @@ packages/gatsby-admin/public
 packages/gatsby/gatsby-admin-public
 packages/gatsby-codemods/transforms
 packages/gatsby-source-graphql/batching
-packages/gatsby-plugin-gatsby-cloud/components
-packages/gatsby-plugin-gatsby-cloud/utils
+packages/gatsby-plugin-gatsby-cloud/**/*.js
```

Comment:
These changes seem unrelated to the PR?

Reply:
I'm not sure about how that got in there. I didn't make that change

Reply:
You'll need to revert those then

Reply:
Cool

Reply:
Reversed the change